shared/actions/tracker.js

```diff
@@ -297,21 +299,45 @@ function _getTrackToken (state, username) {
   return trackerState && trackerState.type === 'tracker' ? trackerState.trackToken : null
 }
 
-export function onFollow (username: string, localIgnore?: bool): (dispatch: Dispatch, getState: () => {tracker: RootTrackerState}) => void {
+function _getUsername (uid: string, getState: () => {tracker: RootTrackerState}): ?string {
```

Comment:
this and related function should be pure, why pass a getState when you can just pass the state data?